src/org/openstreetmap/josm/gui/dialogs/properties/PropertiesDialog.java

```diff
@@ -679,14 +772,18 @@ public void selectionChanged(SelectionChangeEvent event) {
                     ? e.getValue().keySet().iterator().next() : tr("<different>"));
         }
 
+        presets.updatePresets(types, tags, presetHandler);
+    }
+
+    private void updateMembershipTableData(Collection<? extends IPrimitive> primitives) {
         membershipData.setRowCount(0);
 
         Map<IRelation<?>, MemberInfo> roles = new HashMap<>();
-        for (IPrimitive primitive: newSel) {
+        for (IPrimitive primitive: primitives) {
```

Comment:
```suggestion
        for (IPrimitive primitive : primitives) {
```
Since you modified this line. :)